.github/workflows/config/nnx/keras.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
where is this used? from the logic above looks like we are setting NNX via env variable and will always read keras.json based on the backend string (jax), so this will not be used?

Reply:
deleted